Polyfills/Window/Source/Window.cpp

```diff
@@ -10,18 +11,23 @@ namespace Babylon::Polyfills::Internal
         constexpr auto JS_CLASS_NAME = "Window";
         constexpr auto JS_SET_TIMEOUT_NAME = "setTimeout";
         constexpr auto JS_A_TO_B_NAME = "atob";
+
         constexpr auto JS_ADD_EVENT_LISTENER_NAME = "addEventListener";
         constexpr auto JS_REMOVE_EVENT_LISTENER_NAME = "removeEventListener";
+        auto JS_DEVICE_PIXEL_RATIO_NAME = "devicePixelRatio";
     }
 
-    void Window::Initialize(Napi::Env env)
+    void Window::Initialize(Napi::Env env, void* windowPtr)
     {
         Napi::HandleScope scope{env};
 
-        Napi::Function constructor = DefineClass(
+        Napi::Function constructor = Window::DefineClass(
             env,
             JS_CLASS_NAME,
-            {});
+            {
+                StaticAccessor(JS_DEVICE_PIXEL_RATIO_NAME, &Window::GetDevicePixelRatio, &Window::SetDevicePixelRatio)
```

Comment:
As far as I can tell, `SetDevicePixelRatio` is not an API on `window`. Seems like this should be removed.

Reply:
And by removed, I just mean pass in `nullptr` for the mutator function (which is what 99% of our `InstanceAccessor` use cases do).

Reply:
Its not. I'm still addressing some issues with the JS wrapper not properly working, and all of this code is subject to change. Currently the PR is not marked as ready for review, but I appreciate the feedback.

Reply:
Ah ok, I assumed if a PR was published (even in draft) you were looking for feedback. Happy to wait until it is out of draft if that's the process folks tend to use!

Reply:
I'm more interested in large, architectural considerations (Is this even the right problem to solve? are we missing an essential usecase) than small nits (you missed spacing/you don't need this function, etc...) since a lot of this may be rewritten, but thanks